train.py

```diff
@@ -166,7 +166,7 @@ def train(hyp,  # path/to/hyp.yaml or hyp dictionary
         loggers.on_params_update({"batch_size": batch_size})
 
     # Optimizer
-    nbs = 64  # nominal batch size
+    nbs = batch_size*opt.gradient_accum_steps if opt.overwrite_batch_size else 64   # nominal batch size
```

Comment:
Are the changes to this file supposed to be in this PR? They seem unrelated to anchor_grid changes

Reply:
Fixed, thanks for catching that. Those changes are a part of the open PR #90